### PR TITLE
Try to work with alpha.21

### PR DESCRIPTION
**Pull request by pksunkara**

### CODE REVIEW

**Comment by MansoorBashaBellary** [comment]
Hi @pksunkara - Could you please let me know when this PR would be merged. Using latest versions of `storybooks`, for example `4.0.9`, is breaking the CSS of `storybooks` because `vue-cli-plugin-storybook` supports `addons` version of `4.0.0-alpha.20`.

And also, could you please make sure that, `vue-cli-plugin-storybook` works for any version of `storybooks `

**Comment by pksunkara** [comment]
The issue is it's not working. I keep running into a wall. 😞

**Comment by MansoorBashaBellary** [comment]
Oh. Sorry about that.

I worked on deep theming `Stories Nav` panel in `react storybooks`. 

https://github.com/storybooks/storybook/pull/4702

When I tried to use storybooks version to `4.1.0-alpha.2`, Storybooks CSS started breaking in the page, that is because, I am forced to use `@storybook/vue` version to `4.0.0-alpha.25` or else my `storybooks` are breaking.

Thanks for the quick response. :)

File: index.js [hunk 1]
```diff
@@ -57,10 +60,14 @@ module.exports = (api, { pluginOptions = {} }) => {
...
       packageJson: {
         name: '@storybook/vue',
-        version: '4.0.0-alpha.20',
+        version: '^4.0.0',
       },
-      wrapInitialConfig: wrapInitialConfig(api, options),
-      wrapDefaultConfig,
+      frameworkPresets: [
+        {
+          preset: require.resolve('./preset'),
```
Comment (backbone87) [inline index.js]: The key for the preset path should be `name` not `preset`
```suggestion
          name: require.resolve('./preset'),
```

See https://github.com/storybooks/storybook/blob/master/lib/core/src/server/presets.js#L22

**Comment by pksunkara** [comment]
@backbone87 Thanks for the help. I am so embarrassed that I missed that.

File: preset.js [hunk 2]
```diff
@@ -36,4 +36,11 @@ module.exports = {
       resolveLoader: resolvedConfig.resolveLoader,
     };
   },
+  webpackFinal: config => ({
+    ...config,
+    module: {
+      ...config.module,
+      rules: config.module.rules.slice(0, -3),
```
Comment (backbone87) [inline preset.js]: Could you add a comment why you slice off here? what rules are removed by that?

Comment (pksunkara) [reply preset.js]: Vue project's webpack config has css loaders and stuff. But storybooks adds them again which creates an error. This part of code is making sure that it doesn't happen.

**Comment by pksunkara** [comment]
> had to look quite a while myself. does that work now correclty with latest storybook versions?

Yes, released 0.5.0 which should work storybook latest versions.

File: preset.js (hunk 2 above)
Comment (backbone87) [reply preset.js]: Do you remove the Vue ones or the storybook ones? just comment the code, so others know whats goin on here, makes it alot easier to debug problems with missing rules/unexpected rulesets.

Comment (pksunkara) [reply preset.js]: Storybook ones

Comment (pksunkara) [reply preset.js]: https://github.com/storybooks/vue-cli-plugin-storybook/commit/75e9d8a57b0c80fb63f82c616d24e8a12d66aa47

**Pull request merged**